developer/src/kmc-kmn/src/compiler/compiler.ts

```diff
@@ -345,10 +345,12 @@ export class KmnCompiler implements UnicodeSetParser {
     // The compiler detected a .kvks file, which needs to be captured
     kvksFilename = this.callbacks.resolveFilename(kmnFilename, kvksFilename);
     const filename = this.callbacks.path.basename(kvksFilename);
+    let basename = null;
     let vk: VisualKeyboard.VisualKeyboard = null;
     if(filename.endsWith('.kvk')) {
```

Comment:
```suggestion
    if(filename.endsWith(KeymanFileTypes.Binary.VisualKeyboard)) {
```

This should have been done before... oops my bad.